Cartfile.resolved

```diff
@@ -1,4 +1,4 @@
 github "Boris-Em/BEMCheckBox" "1.4.1"
-github "JSONModel/JSONModel" "364f57ba9671af417b35762049d8aed482fcf075"
-github "jdg/MBProgressHUD" "1.0.0"
-github "mxcl/PromiseKit" "4.3.2"
+github "JSONModel/JSONModel" "cebb6d9cc1320eb20b61adea31a0c894b3cb7266"
```

Comment:
Don't think we want this file added? I think a `pod install` updated it

Reply:
It's just upgrading the pinned versions. This is fine.